lib/simp/cli/config/items/data/simp_options_puppet_ca_port.rb

```diff
@@ -6,9 +6,18 @@ module Simp::Cli::Config
   class Item::SimpOptionsPuppetCAPort < IntegerItem
     def initialize
       super
+
+      @port = 8141
+      if defined?(Pry)
```

Comment:
why is this debug here?

Reply:
Must have been left over from the rebase